lmcache/v1/storage_backend/connector/instrumented_connector.py

```diff
@@ -48,7 +48,9 @@ async def put(self, key: CacheEngineKey, memory_obj: MemoryObj) -> None:
         end = time.perf_counter()
         self._stats_monitor.update_interval_remote_time_to_put((end - begin) * 1000)
         self._stats_monitor.update_interval_remote_write_metrics(obj_size)
-        logger.debug(f"Bytes offloaded: {obj_size / 1e6:.4f} MBytes")
+        logger.debug(
```

Comment:
Think twice, would you like to help improve this block to avoid extra cost while log level is info as normal.
- Use lambda for lazy calculate value only while log level higer than debug. e.g.
```python
logger.debug("Bytes offloaded: %s in %s", 
                     lambda: fmt_data_size(obj_size), 
                     lambda: fmt_time_took(begin, end))
```
- Add a condition check directly.
```python
if logger.isEnabledFor(logging.DEBUG):
    xxxx
```